simplq/src/components/ErrorHandler.jsx

```diff
@@ -23,7 +21,6 @@ export class ErrorBoundary extends React.Component {
   componentDidCatch(error, errorInfo) {
     // You can also log the error to an error reporting service
     // logErrorToMyService(error, errorInfo);
-    store.dispatch(setErrorPopupMessage('An error occured. Please try again'));
```

Comment:
Should we be keeping it, or not? 🤔 

What's your points for removing it?

Reply:
I figured it wasn't needed since we were redirecting to the 404 page anyway. It was also never showing up when I tested it, so that'll take some debugging. Do you think we should keep it?